flow/alerting/classifier.go

```diff
@@ -338,6 +338,11 @@ func GetErrorClass(ctx context.Context, err error) (ErrorClass, ErrorInfo) {
 				return ErrorNotifyConnectivity, pgErrorInfo
 			}
 
+			// Handle Neon's custom WAL reading error
+			if strings.Contains(pgErr.Message, "server closed the connection unexpectedly") && pgErr.Routine == "NeonWALPageRead" {
```

Comment:
should flip comparison so Contains code runs less often

Reply:
oops, had this code on auto-merge -- handled in https://github.com/PeerDB-io/peerdb/pull/3422